### DISCUSSION TITLE
Multiple applications logged in one instance of seq

### DISCUSSION THREAD

**Discussion by johnhoge** [datalust/seq-tickets#2286] [Ideas]
Is there a nice clean way to log from multiple apps into Seq? I know there is a 3 instance pack license but that would involve logging in to all 3 on a regular basis to see if there were any issues. I'd rather just log in to once and see if there are any issues with any of my apps.

**Comment by nblumhardt** [comment]
Hi @johnhoge; yes, Seq's designed to accept logs from multiple apps into a single, central instance.

To keep track of which events are from which app, the best approach is to create a unique API key for each app, and use the "attached properties" option to mark ingested events with an application name.

https://docs.datalust.co/docs/api-keys

Once the events are ingested, you can create signals based on the application name to filter down to just one app or another:

https://docs.datalust.co/docs/signals

Hope this helps,
Nick

**Comment by johnhoge** [comment]
This is great news thanksSent from my iPadOn Sep 28, 2024, at 5:25 PM, Nicholas Blumhardt ***@***.***> wrote:﻿
Hi @johnhoge; yes, Seq's designed to accept logs from multiple apps into a single, central instance.
To keep track of which events are from which app, the best approach is to create a unique API key for each app, and use the "attached properties" option to mark ingested events with an application name.
https://docs.datalust.co/docs/api-keys
Once the events are ingested, you can create signals based on the application name to filter down to just one app or another:
https://docs.datalust.co/docs/signals
Hope this helps,
Nick

**Comment by johnhoge** [comment]
Hi Nick,

Never mind my last email I forgot to set the attached properties.

All set now. Thanks for your help.
________________________________
